data/json/items/gun/40.json

```diff
@@ -207,5 +207,61 @@
       [ "stock", 1 ],
       [ "underbarrel", 1 ]
     ]
+  },
+  {
+    "id": "hi_power_40",
+    "copy-from": "pistol_base",
+    "type": "GUN",
+    "name": "Browning Hi-Power .40 S&W",
+    "description": "The Browning Hi-Power is a semi automatic handgun. It is one of the most widely used military pistols in history. This can be loaded with .40 S&W.",
```

Comment:
```suggestion
    "description": "The Browning Hi-Power is a semi-automatic handgun developed shortly before the second world war.  Widely issued since then, it remains in use by India, Canada and Australia.  This is a commercial variant produced by Browning Arms in .40 S&W.",
```